### PR TITLE
Cloud changelog - Apr 27

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by marianunez — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -4,12 +4,51 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added new API endpoints ``GET /api/v4/teams/:team_id/top/reactions`` and ``GET /api/v4/users/me/top/reactions`` to get top reactions for a team and user.
+ - Bumped Go to v1.18.
+
+#### Performance
+ - Improved the performance of ``GetTeamsUnreadForUser`` when Collapsed Reply Threads is enabled.
+ - Improved the performance of configuration read/writes if the configuration is stored on a database.
+
+#### Bug Fixes
+ - Fixed an issue where the ``UpdateUser`` API endpoint required a ``create_at`` field.
+ - Fixed an issue where permalinks to Direct and Group Message posts did not show a preview.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:37]: Docs is moving away from capitalizing product features like Direct Messages or Group Messages.

Comment (cwarnermm) [inline source/install/cloud-changelog.md:37]:
```suggestion
 - Fixed an issue where permalinks to direct and group message posts did not show a preview.
```

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -4,12 +4,51 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where a mention badge was present when marking a root post with a mention as unread on the right-hand side with Collapsed Reply Threads enabled.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:38]: I'm not clear on what we've changed/fixed in this situation. Have we hidden the mention badge under these circumstances?

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -4,12 +4,51 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where copy links did not fire a track callback until the copy link was actually clicked.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:39]: What would this look like to an end user who is unfamiliar with track callbacks?

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -4,12 +4,51 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where the public link to generate the API was getting called even if public links were disabled.
+ - Fixed an issue with onboarding page view events.
+ - Fixed an issue where the custom emoji "Next" button was out of view when a banner was present.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:42]:
```suggestion
 - Fixed an issue where the custom emoji **Next** button was out of view when a banner was present.
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

File: source/install/cloud-changelog.md (hunk 2 above)
Comment (amyblais) [reply source/install/cloud-changelog.md]: I clarified the text.

File: source/install/cloud-changelog.md (hunk 3 above)
Comment (amyblais) [reply source/install/cloud-changelog.md]: I removed this as it's likely not important to add here.

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+### Improvements
+
+#### User Interface (UI)
+ - Added the channel members list to the right-hand side Channel Info modal.
+ - Added the ability to invite new users to a team from the **Add to channel** modal.
+ - To be able to download images and copy public links for images quicker, a copy URL and download buttons were added to image thumbnails.
+ - Added the ability to have one-character long channel names.
+
+#### Administration
+ - Added a new config cleanup job to regularly remove outdated entries from the database. The threshold for this setting can be adjusted with ``JobSettings.CleanupConfigThresholdDays``.
```
Comment (agnivade) [inline source/install/cloud-changelog.md:25]: Added a new cleanup job to regularly remove outdated config entries from the database. The threshold ..

**Review by agnivade — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+### Improvements
+
+#### User Interface (UI)
+ - Added the channel members list to the right-hand side Channel Info modal.
+ - Added the ability to invite new users to a team from the **Add to channel** modal.
+ - To be able to download images and copy public links for images quicker, a copy URL and download buttons were added to image thumbnails.
+ - Added the ability to have one-character long channel names.
+
+#### Administration
+ - Added a new cleanup job to regularly remove outdated config entries from the database. The threshold for this setting can be adjusted with ``JobSettings.CleanupConfigThresholdDays``.
```
Comment (jasonblais) [inline source/install/cloud-changelog.md:25]: Not for Cloud

File: source/install/cloud-changelog.md [hunk 7]
```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added a new API endpoint ``POST /api/v4/users/{user_id}/teams/{team_id}/threads/{thread_id}/set_unread/{post_id}`` to set a thread as unread by post id.
+ - Added ``fromWebhook`` property to the webapp plugin API.
+ - Added new API endpoints ``GET /api/v4/teams/:team_id/top/reactions`` and ``GET /api/v4/users/me/top/reactions`` to get top reactions for a team and user.
+ - Bumped Go to v1.18.
```
Comment (jasonblais) [inline source/install/cloud-changelog.md:29]: Not for Cloud

File: source/install/cloud-changelog.md [hunk 8]
```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+#### User Interface (UI)
+ - Added the channel members list to the right-hand side Channel Info modal.
+ - Added the ability to invite new users to a team from the **Add to channel** modal.
+ - To be able to download images and copy public links for images quicker, a copy URL and download buttons were added to image thumbnails.
+ - Added the ability to have one-character long channel names.
+
+#### Administration
+ - Added a new cleanup job to regularly remove outdated config entries from the database. The threshold for this setting can be adjusted with ``JobSettings.CleanupConfigThresholdDays``.
+ - Added a new API endpoint ``POST /api/v4/users/{user_id}/teams/{team_id}/threads/{thread_id}/set_unread/{post_id}`` to set a thread as unread by post id.
+ - Added ``fromWebhook`` property to the webapp plugin API.
```
Comment (jasonblais) [inline source/install/cloud-changelog.md:27]: Not for Cloud

File: source/install/cloud-changelog.md [hunk 9]
```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+#### User Interface (UI)
+ - Added the channel members list to the right-hand side Channel Info modal.
+ - Added the ability to invite new users to a team from the **Add to channel** modal.
+ - To be able to download images and copy public links for images quicker, a copy URL and download buttons were added to image thumbnails.
+ - Added the ability to have one-character long channel names.
+
+#### Administration
+ - Added a new cleanup job to regularly remove outdated config entries from the database. The threshold for this setting can be adjusted with ``JobSettings.CleanupConfigThresholdDays``.
+ - Added a new API endpoint ``POST /api/v4/users/{user_id}/teams/{team_id}/threads/{thread_id}/set_unread/{post_id}`` to set a thread as unread by post id.
```
Comment (jasonblais) [inline source/install/cloud-changelog.md:26]: For this and the other API change that remains, I would change this section header to `#### API changes for Custom Integrations`

File: source/install/cloud-changelog.md [hunk 10]
```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added ``fromWebhook`` property to the webapp plugin API.
+ - Added new API endpoints ``GET /api/v4/teams/:team_id/top/reactions`` and ``GET /api/v4/users/me/top/reactions`` to get top reactions for a team and user.
+ - Bumped Go to v1.18.
+
+#### Performance
+ - Improved the performance of ``GetTeamsUnreadForUser`` when Collapsed Reply Threads is enabled.
+ - Improved the performance of configuration read/writes if the configuration is stored on a database.
```
Comment (jasonblais) [inline source/install/cloud-changelog.md:33]: Not relevant for Cloud

File: source/install/cloud-changelog.md [hunk 11]
```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+#### Bug Fixes
+ - Fixed an issue where the ``UpdateUser`` API endpoint required a ``create_at`` field.
```
Comment (jasonblais) [inline source/install/cloud-changelog.md:36]: I would move this to the above API changes section

**Review by jasonblais — CHANGES_REQUESTED** [review]

**Review by jasonblais — APPROVED** [review]

**Review by spirosoik — APPROVED** [review]

**Review by dontoisme — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 12]
```diff
@@ -4,12 +4,46 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added a new API endpoint ``POST /api/v4/users/{user_id}/teams/{team_id}/threads/{thread_id}/set_unread/{post_id}`` to set a thread as unread by post id.
+ - Added new API endpoints ``GET /api/v4/teams/:team_id/top/reactions`` and ``GET /api/v4/users/me/top/reactions`` to get top reactions for a team and user.
+ - Fixed an issue where the ``UpdateUser`` API endpoint required a ``create_at`` field.
+
+#### Performance
+ - Improved the performance of ``GetTeamsUnreadForUser`` when Collapsed Reply Threads is enabled.
+
+#### Bug Fixes
+ - Fixed an issue where permalinks to direct and group message posts did not show a preview.
+ - Fixed an issue where a mention badge was unexpectedly present in addition to the thread item bolding when marking a root post with a mention as unread on the right-hand side with Collapsed Reply Threads enabled.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:34]:
```suggestion
 - Fixed an issue when Collapsed Reply Threads are enabled where marking a root post with a mention as unread displayed both a mention badge and the thread item being bolded.
```

**Review by cwarnermm — COMMENTED** [review]
Thanks! Feedback provided inline.

**Review by dontoisme — APPROVED** [review]

**Review by esethna — APPROVED** [review]

**Pull request merged**